### PR TITLE
feat: Add browsing description

### LINKED ISSUE #247: Browse datasets by topic - Adding a descriptive text
In order to give the user a bit more guidance, of what he can do, on the dataset page, a descriptive text should be added below the title.

Screenshot 2021-12-02 at 13 43 14

### PR DESCRIPTION
**Pull request by ptbrowne**
Adding a descriptive text on the browsing datasets index page.

### CODE REVIEW

File: app/configurator/components/select-dataset-step.tsx [hunk 1]
```diff
@@ -113,16 +113,41 @@ export const SelectDatasetStepContent = () => {
...
-                .map((f) => (f.__typename !== "DataCubeAbout" ? f.label : null))
-                .join(", ")
-            ) : (
-              <Trans id="browse.datasets.all-datasets">All datasets</Trans>
-            )}
-          </Text>
+          {dataset ? null : filters.length > 0 ? (
+            filters
+              .filter((f) => f.__typename !== "DataCubeAbout")
+              .map((f) => (f.__typename !== "DataCubeAbout" ? f.label : null))
```
Comment (bprusinowski) [inline app/configurator/components/select-dataset-step.tsx:119]: Do we need this condition in .map? Maybe a type guard in filter would work?

Comment (ptbrowne) [reply app/configurator/components/select-dataset-step.tsx]: Ah, it worked finally: c0a7315

Comment (bprusinowski) [reply app/configurator/components/select-dataset-step.tsx]: Yes, I was testing the same approach 😅

**Review by bprusinowski — APPROVED** [review]
LGTM!

**Pull request merged**